docs/src/index.md

```diff
@@ -0,0 +1,16 @@
+# RandomFeatures
+
+A julia package to construct and apply random feature methods for regression.
+RandomFeatures can be viewed as an approximation of kernel methods. They can be used both as a substitution in Kernel ridge regression and Gaussian Process regresison. 
+
+
+## Highlights
```

Comment:
Can skip this on a first pass, but think index.md conventionally has a table listing modules and a one-line summary of their purpose (along the lines of the walkthrough you gave me on Tues).

Reply:
Done anyway, it only took a minute I've kept it brief though until things are set!